browser/ui/tabs/split_view_tab_strip_model_adapter.cc

```diff
@@ -30,7 +30,7 @@ SplitViewTabStripModelAdapter::~SplitViewTabStripModelAdapter() = default;
 void SplitViewTabStripModelAdapter::MakeTiledTabsAdjacent(
     const SplitViewBrowserData::Tile& tile,
     bool move_right_tab) {
-  auto [tab1, tab2] = tile;
+  auto [tab1, tab2, _] = tile;
```

Comment:
Curious there is a way to make this ok when tile has more than 3 vars.

Reply:
Yeah, it's not scalable. I think we might want replace this with 

```
tab1 = tile.first
tab2 = tile.second
```